keyboards/handwired/twadlee/tp69/config.h

```diff
@@ -41,12 +41,39 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
  */
 /*                        19  17  16  15  14   2   8   6   7   9 */
-#define MATRIX_ROW_PINS { B2, B1, B0, C0, D1, D0, D3, D4, D2, C3 }
+//#define MATRIX_ROW_PINS { B2, B1, B0, C0, D1, D0, D3, D4, D2, C3 }
 /*                        20  21  18    0    1   3   4   5 */
-#define MATRIX_COL_PINS { D5, D6, A4, B16, B17, A1, A2, D7 }
+//#define MATRIX_COL_PINS { D5, D6, A4, B16, B17, A1, A2, D7 }
+#define MATRIX_ROW_PINS {                       \
```

Comment:
Is there a reason to move away from the regular pin notation here?

Reply:
Not especially. Just seemed more convenient to use pin names that mapped directly to the labels on the physical unit.